test/test_corpus.cc

```diff
@@ -495,6 +495,129 @@ TEST_P(ExpectationTest, PerPhaseTest) { // NOLINT
     // Allow later phases to have errors that we didn't test for
     errorQueue->drainAllErrors();
 
+    // now we test the incremental resolver
+
+    auto disableStressIncremental =
+        BooleanPropertyAssertion::getValue("disable-stress-incremental", assertions).value_or(false);
+    auto isAutogenTest = test.expectations.find("autogen") != test.expectations.end();
+    if (disableStressIncremental || isAutogenTest) {
+        TEST_COUT << "errors OK" << '\n';
```

Comment:
If you put the below logic into a helper function, you could just do:

```c++
if (!disableStressIncremental && !isAutogenTest) {
  runStressIncrementalResolverTest(...);
}
TEST_COUT << /* etc */
```

If it's unwieldy because it needs a lot of arguments, then it's fine to keep inline.

Reply:
Ah, and you'll need to `ASSERT_NO_FATAL_FAILURE(runStressIncrementalResolverTest(...));` if you want tests to early exit on an ASSERT failure in `runStressIncrementalResolverTest`, since gtest doesn't use exceptions.